README.md

```diff
@@ -61,15 +71,18 @@ CONNECTOR_LEDGERS={
     "currency": "EUR",
     "plugin": "ilp-plugin-payment-channel-framework",
     "options": {
-      "prefix": "g.eur.mytrustline.",
-      "token": "shared_secret", // shared secret between server and client
-      "rpcUri": "https://wallet1.example/api/peers/rpc",
+      "server": "btp+wss://wallet1.example:1234/example_path",
+      "token": "shared_secret"
```

Comment:
We decided to use the "btp+wss://auth_username:auth_token@wallet1.example:1234/example_path" format for putting server and token in one URI. If you're not going to use that, then maybe remove the 'btp+' prefix, rename token to authToken, and add authUsername.

Reply:
I fixed this in https://github.com/interledgerjs/ilp-plugin-payment-channel-framework/pull/27